CMakeLists.txt

```diff
@@ -29,6 +29,7 @@ add_custom_target(dist COMMAND ${CMAKE_MAKE_PROGRAM} package_source)
 
 # Options
 option(BUILD_MAN_PAGES "Build man pages" OFF)
+option(INSTALL_SDDM_CONF "Install sddm.conf" OFF)
```

Comment:
I'd add a slightly longer description - `"Install sddm.conf containing current default values for all configuration options"` or similiar.